buildpipeline/DotNet-CoreRT-Linux.json

```diff
@@ -503,6 +539,60 @@
     {
       "enabled": true,
       "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Download nuget",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "curl",
+        "arguments": "-L -k -o $(SourceFolder)/nuget.zip https://dotnet.myget.org/F/dotnet-buildtools/api/v2/package/NuGet.CommandLine/3.5.0-rc-1256",
+        "workingFolder": "",
+        "failOnStandardError": "false"
+      }
+    },
+    {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Unzip nuget",
+      "timeoutInMinutes": 0,
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "unzip",
+        "arguments": "$(SourceFolder)/nuget.zip -d $(SourceFolder)/nuget",
+        "workingFolder": "",
+        "failOnStandardError": "false"
+      }
+    },
+    {
+      "enabled": false,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "Publish packages",
```

Comment:
Thoughts on following the corefx/coreclr/wcf convention? (Where packages from each leg are published onto Azure blob storage, and once all legs are done a single build pulls down all packages and publishes them.) It makes it simple to publish the package list to the versions repo, and gets rid of the duplicated effort of getting nuget.

Changing to that flow could be done later, though.